test/apps/widgets-example-9.2/app.ts

```diff
@@ -103,9 +105,22 @@ const deck = new Deck({
     new _ScaleWidget({placement: 'bottom-right'}),
     new _GeolocateWidget({viewId: 'left-map'}),
     new _ThemeWidget(),
-    new _InfoWidget({mode: 'hover', getTooltip}),
-    new _InfoWidget({mode: 'click', getTooltip}),
-    // new _InfoWidget({mode: 'static', getTooltip})
+    new _ContextMenuWidget({
+      getMenuItems: (info: PickingInfo) => {
+        const name = info.layer?.id === 'airports' && info.object?.properties.name;
+        console.log('Context menu:', name);
+        return (
+          name && [
+            {key: 'airport', label: `${name}`},
+            {key: 'open', label: 'Open in new tab'},
+            {key: 'favorite', label: 'Set as favorite'},
+            {key: 'filter', label: 'Exclude from filter'}
+          ]
+        );
+      }
+    }),
+    // new _InfoWidget({mode: 'hover', getTooltip}),
+    // new _InfoWidget({mode: 'click', getTooltip}),
```

Comment:
Why comment these out?

Reply:
> Why comment these out.
They didn't seem to be working at the moment, needs a fix.